src/blocks/ImageTextBlock.tsx

```diff
@@ -158,16 +179,51 @@ export const ImageTextBlock: React.FunctionComponent<ImageTextBlockProps> = ({
   size,
   media_position,
   button_color,
+  button_size,
   button_weight,
+  button_position_mobile,
 }) => {
+  const getButton = () =>
```

Comment:
i think we should move this out to be a "normal" react component :) so it's used like
```jsx
<AlignedButton branchLink={button_branch_link} size={button_size} ...blablabla />
```
 or something like that